packages/trader/src/Assets/Trading/Categories/accumulator-trade-description.jsx

```diff
@@ -1,59 +1,41 @@
-            ),
-        },
-        { type: 'heading', text: localize('Maximum ticks') },
-        {
-            type: 'paragraph',
-            text: localize(
-                'We’ve also limited the maximum duration for every contract, and it differs according to the accumulator value that you choose. Your contract will be closed automatically when the maximum duration is reached.'
-            ),
-        },
-    ];
-
+const AccumulatorTradeDescription = ({ onClick }) => {
     return (
         <React.Fragment>
-            {content.map(({ type, text }, index) =>
-                /* We're using the array’s indexes as keys in this “static” list, which is never re-ordered, for better performance. https://www.developerway.com/posts/react-key-attribute
-            We're not using text itself as keys here because text will update each time platform language is changed. */
-                type === 'heading' ? (
-                    <Text as='h2' key={index} weight='bold'>
-                        {text}
-                    </Text>
-                ) : (
-                    <Text as='p' key={index}>
-                        {text}
-                    </Text>
-                )
-            )}
+            <Text as='p'>
+                <Localize
+                    i18n_default_text='Accumulators allow you to express a view on the range of movement of an index and grow your stake exponentially at a fixed <0>growth rate</0>.'
+                    components={[<span className='contract-type-info__content-definition' onClick={onClick} key={0} />]}
+                />
+            </Text>
+            <Text as='p'>
+                <Localize
+                    i18n_default_text='Your <0>payout</0> is the sum of your inital stake and profit.'
+                    components={[<span className='contract-type-info__content-definition' onClick={onClick} key={0} />]}
+                />
+            </Text>
+            <Text as='p'>
+                <Localize
+                    i18n_default_text='Your stake will continue to grow as long as the current spot price remains within a specified <0>range</0> from the <0>previous spot price</0>. Otherwise, you lose your stake and the trade is terminated.'
+                    components={[<span className='contract-type-info__content-definition' onClick={onClick} key={0} />]}
+                />
+            </Text>
+            <Text as='p'>
+                <Localize
+                    i18n_default_text='You can close your trade anytime. However, be aware of <0>slippage risk<0/>.'
+                    components={[<span className='contract-type-info__content-definition' onClick={onClick} key={0} />]}
+                />
+            </Text>
         </React.Fragment>
     );
 };
+
+AccumulatorTradeDescription.PropTypes = {
+    onClick: PropTypes.func,
+};
```

Comment:
@henry-deriv let's make this file .tsx in order to avoid conflict with https://github.com/binary-com/deriv-app/pull/8852/